drivers/input/touchscreen/tsc2007_core.c

```diff
@@ -234,29 +234,20 @@ static int tsc2007_probe_properties(struct device *dev, struct tsc2007 *ts)
 	u32 val32;
 	u64 val64;
 
-	if (!device_property_read_u32(dev, "ti,max-rt", &val32))
+	if (!device_property_read_u32(dev, "touchscreen-max-pressure", &val32))
```

Comment:
`touchscreen-max-pressure` is read by [touchscreen_parse_properties](https://elixir.bootlin.com/linux/latest/C/ident/touchscreen_parse_properties). Is there a need to read it in two locations?

Set the default to MAX_12BIT before calling `touchscreen_parse_properties`, and it will overwrite that default if found.